### ISSUE TITLE
-parallel-testing-enabled flag always set to "NO"

### ISSUE THREAD

**Issue by nathan-149** [lyndsey-ferguson/fastlane-plugin-test_center#247]
### New Issue Checklist

- [x] Updated `fastlane-plugin-test_center` to the latest version
- [x] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [x] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [x] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

> If you love this fastlane plugin, consider sponsoring it or asking your company to sponsor it. I would really appreciate any
> gesture: https://github.com/sponsors/lyndsey-ferguson. 😍

### Issue Description

multi_scan

I am trying to run multi_scan:
  lane :multi_scan_unit_tests do
    begin
      multi_scan(
        workspace: File.absolute_path([Workspace Path]),
        scheme: '[Scheme Name]',
        try_count: 3,
        fail_build: false,
        parallel_testrun_count: 3,
        output_directory: "reports/test_report",
        include_simulator_logs: false,
        skip_build: true,
        code_coverage: false,
        skip_slack: true,
        clean: false,
        xctestrun: "[path].xctestrun",
      )
    end
  end

It tests completely fine, but has the flag -parallel-testing-enabled flag always set to NO in the xcargs. Even when I specify xcargs to set it to YES, it shows up as NO when the tests starts. So, only tests on one simulator, instead of the expected 3 simulators. 

Any help would be greatly appreciated! :)

**Comment by lyndsey-ferguson** [comment]
Please provide this info as suggested by the placeholder text that was present when this Issue was first created:
```
##### Complete output when running fastlane, including the stack trace and command used
<!-- You can use: `--verbose --capture_output` as the last commandline arguments to get that collected for you -->

<!-- The output of `--verbose --capture_output` could contain sensitive data such as application ids, certificate ids, passwords, authentication tokens, or email addreses, Please make sure you double check the output and replace anything sensitive you don't wish to submit in the issue -->

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>

### Environment

<!-- Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used the `--capture_output` option, please remove this block as it is already included there. -->

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>
```

Note that `-parallel-testing-enabled` feature is meant to run 3 simulators for the same batch of tests, which interferes with the goal of `:parallel_testrun_count`, which is to divide all your tests into 3 batches and run each batch of tests on its own simulator, in parallel. 

It is intentional that `-parallel-testing-enabled` is set to NO.

**Comment by nathan-149** [comment]
I see, that makes a lot of sense to me. Thanks so much! 
However, when I set parallel_testrun_count to 4, it still only completes all tests on one simulator. Is this unexpected?

**Comment by lyndsey-ferguson** [comment]
You should see 4 simulators running and the logs in the console indicating which "worker" it ran on....unless you don't have even 4 tests to run.

```bash
[worker 1][timestamp] log message 
[worker 1][timestamp] log message 
[worker 1][timestamp] log message 
[worker 2][timestamp] log message 
[worker 2][timestamp] log message 
[worker 2][timestamp] log message 
```

**Comment by lyndsey-ferguson** [comment]
Please include the logs and I may be able to provide more info/help.

**Comment by nathan-149** [comment]
Thanks for the reply. The output right now doesn't show any workers for me (not even [worker 1]), just standard xcpretty output. Unfortunately I can't include logs because this is for a large company pipeline - thank you though, I'll look more into it!

**Comment by lyndsey-ferguson** [comment]
Are you comfortable with Ruby? I can offer you some guidance on how to debug it.

**Comment by lyndsey-ferguson** [comment]
One thing I would suggest is to use the `--verbose` flag, and watch for any weird issues. 

Keep in mind that you can simply remove sensitive info by renaming tests, targets, paths, etc.

**Comment by nathan-149** [comment]
Thanks for being so helpful!!! I'm not that familiar with Ruby, but will try my best to debug it for now, and let you know soon! :)

**Comment by lyndsey-ferguson** [comment]
@nathan-149 Okay, good luck! I suggest that you take a look at [the debugging guidelines](https://github.com/fastlane/fastlane/blob/master/ToolsAndDebugging.md) put out by fastlane to start. That's how I started learning and contributing to fastlane (and then creating plugins).

I'm going to close this issue as I cannot reproduce it and you're not authorized to share info that will help.
